packages/ckeditor5-ckbox/src/ckboxcommand.ts

```diff
@@ -387,6 +389,25 @@ function getAssetUrl( asset: CKBoxRawAssetDefinition ) {
 	return url.toString();
 }
 
+/**
+ * Focuses the CKBox dialog.
+ *
+ * @param wrapper The element in DOM which wrap CKBox component.
+ */
+function focusDialog( wrapper: Element | null ) {
+	// If the DOM is not fully loaded and rendered by the time the focus() method is invoked,
+	// the element might not get focused. So we call the focus() when the thread becomes idle.
+	setTimeout( () => {
+		// Wrapper cannot be focused by default because it's covered by it's children entirely.
+		// Then we getting to the nearest focusable child( dialog container ) inside wrapper.
+		const dialogContainer = wrapper && wrapper.children[ 0 ] && wrapper.children[ 0 ].children[ 0 ];
```

Comment:
This is a very unstable way of selecting the element. You don't have any control over the component you are integrating and with such binding any change in its structure that you won't be aware of (like adding an extra wrapper) will break your code.

Unfortunately I don't think the component exposes some parts through it's API, but plugging into an element with a class `ckbox-gallery` seems to be much safer (it will work no matter where it will be moved unless the class name changes - but it looks pretty generic, so I think it shouldn't). And actually I'd focus it's first child - the first image in the gallery (as I think that's the most common intention when opening the dialog).